### DISCUSSION TITLE
iOS capacitor app not working on device (black screen), but working in simulator

### DISCUSSION THREAD

**Discussion by Dylan190774** [quasarframework/quasar#14553] [CLI - Capacitor / Cordova modes] [answered]
I'm trying to get Quasar with Capacitor to work on iOS. 
Working on MacOS, I updated all CLI tools (quasar, yarn, ios-deploy, etc.) to the latest version, and create a default test app with `yarn create quasar` with all default options.

When I try to run it in Xcode, the app works in the iPhone-simulator, but when I try to deploy to a real device, I only see a black screen and nothing happens.
The device is an old iPhone 5s with iOS 12.5.6. Is this too old or should this work?

UPDATE
I think I found what is causing this. I checked the Axios box while creating the app.
`Axios` uses the function `Promise.allSettled`, which according to [www.caniuse.com](url) can only be used since iOS 13.

Does this mean I can't use Axios, or is there maybe a workaround?

**Comment by Dylan190774** [comment, marked as answer]
The solution is to change the `browserslist `entries in `package.json` : 

`"last 15 iOS versions"`

See [this page](https://browsersl.ist/#q=last+15+iOS+versions) for what this does.

This will make the compiler use some polyfills to include some newer functions.

**Comment by rstoenescu** [comment]
@Dylan190774 
Backward compatibility with older webviews will be available out of the box in q/app-vite 1.1.3 and q/app-webpack 3.6.2
Just pushed the changes
